### PR TITLE
Migrate bank account approvals page

### PR DESCRIPTION
**Pull request by AndyClifford**
**Ticket:** https://www.notion.so/centrapay/Lanterns-d784e8a5f9fa44328a5d0ae1fd85d37e?p=c6d64de29a2245358a5e9b7eed5edbcc&pm=s

### CODE REVIEW

File: astro.config.mjs [hunk 1]
```diff
@@ -39,5 +39,9 @@ export default defineConfig({
   redirects: {
     '/guides/merchant-terminals/': '/guides/payment-flows/',
     '/accounts/': '/api/accounts/',
+    '/fiat/': '/api/bank-accounts/',
```
Comment (AndyClifford) [inline astro.config.mjs:42]: These were supposed to be included in the last PR

File: src/content/api/bank-account-approvals.mdx [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+  <Property name="id" type="string">
+    The Bank Account Approval's unique identifier.
+  </Property>
+
+  <Property name="bankAccountId" type="string">
+    The id of the associated [Bank Account](/api/bank-accounts).
+  </Property>
+
+  <Property name="accountId" type="string">
+    The id of the owning Centrapay [Account](/api/bank-accounts).
```
Comment (alexBinns) [inline src/content/api/bank-account-approvals.mdx:33]: The link is wrong here

Comment (AndyClifford) [reply src/content/api/bank-account-approvals.mdx:33]: The accounts page has already been merged in so I think this one is all good

File: src/content/api/bank-account-approvals.mdx [hunk 3]
```diff
@@ -0,0 +1,374 @@
...
+  <Property name="modifiedAt" type="timestamp">
+    When the Bank Account Approval was updated.
+  </Property>
+
+  <Property name="modifiedBy" type="crn">
+    The User or API Key that updated the Bank Account Approval.
+  </Property>
+
+  <Property name="approvalActivities" type="array">
+    An array of [Bank Account Approval Activity] associated with the Bank Account Approval.
```
Comment (MeganSteenkamp) [inline src/content/api/bank-account-approvals.mdx:64]:
```suggestion
    An array of [Bank Account Approval Activities](#bank-account-approval-activity-model) associated with the Bank Account Approval.
```

File: src/content/api/bank-account-approvals.mdx [hunk 4]
```diff
@@ -0,0 +1,374 @@
...
+  </Properties>
+
+  ### Errors
+  <Properties>
+    <Property name="403" type="APPROVAL_ALREADY_IN_PROGRESS" error>
+      There is already a Bank Account Approval in progress that is awaiting review from Centrapay.
+    </Property>
+  </Properties>
+
+  <CodePanel slot="code-examples" title="Request" method="POST" path="/api/accounts/{accountId}/members">
```
Comment (MeganSteenkamp) [inline src/content/api/bank-account-approvals.mdx:148]:
```suggestion
  <CodePanel slot="code-examples" title="Request" method="POST" path="/api/bank-account-approvals">
```

File: src/content/api/bank-account-approvals.mdx [hunk 5]
```diff
@@ -0,0 +1,374 @@
...
+  path="/api/bank-account-approvals"
+>
+  ## Request Bank Account Approval <Badge type="experimental"/>
+
+  This endpoint allows you to request a Bank Account Approval.
+
+  ### Attributes
+  <Properties>
+    <Property name="mediaUploadId" type="string" required>
+      The id of the associated [Media Upload][/api/media-uploads].
```
Comment (MeganSteenkamp) [inline src/content/api/bank-account-approvals.mdx:129]:
```suggestion
      The id of the associated [Media Upload](/api/media-uploads).
```

File: src/content/api/bank-account-approvals.mdx [hunk 6]
```diff
@@ -0,0 +1,374 @@
...
+    </Property>
+
+    <Property name="type" type="string" required>
+      The [Bank Account Approval Type](#bank-account-approval-types). A Bank Account Approval can only be requested for type `settlement`.
+    </Property>
+  </Properties>
+
+  ### Errors
+  <Properties>
+    <Property name="403" type="APPROVAL_ALREADY_IN_PROGRESS" error>
```
Comment (MeganSteenkamp) [inline src/content/api/bank-account-approvals.mdx:143]: Can you and @alexBinns please catch up on this - IMO it should be a different component or at least take different props as the property names don't really make sense 🤔

File: src/content/api/bank-account-approvals.mdx [hunk 7]
```diff
@@ -0,0 +1,374 @@
...
+  No attributes.
+
+  <CodePanel slot="code-examples" title="Request" method="GET" path="/api/bank-account-approvals/{bankAccountApprovalId}">
+    ```bash
+    curl https://service.centrapay.com/api/bank-account-approvals/bbab9a768921019cb856 \
+      -H "X-Api-Key: $api_key"
+    ```
+  </CodePanel>
+
+  <CodePanel slot="code-examples" title="Request" method="GET" path="/api/bank-account-approvals/{bankAccountApprovalId}">
```
Comment (MeganSteenkamp) [inline src/content/api/bank-account-approvals.mdx:200]:
```suggestion
  <CodePanel slot="code-examples" title="Response">
```

File: src/content/api/bank-account-approvals.mdx (hunk 6 above)
Comment (AndyClifford) [reply src/content/api/bank-account-approvals.mdx:143]: Yeah another component makes sense - I was thinking we would just fix forward on this as the syntax will be almost identical

Comment (AndyClifford) [reply src/content/api/bank-account-approvals.mdx:142]: Megan I've made a component for this in https://github.com/centrapay/centrapay-docs/pull/825 and have backfilled existing migrations

**Review by MeganSteenkamp — APPROVED** [review]

**Pull request merged**